jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -43,8 +43,11 @@
 
 	private static void init() {
```

Comment:
we should simply put ifdefs around loadlibrary and set `isCRIUSupportEnabled` if criu is not enabled

Reply:
good idea, decorated the code in question w/ JPP flag `CRIU_SUPPORT`.